lib/promisepay_ex/client.ex

```diff
@@ -66,7 +66,15 @@ defmodule PromisepayEx.Client do
         }
       } ->
         {:ok, {401, headers, errors}}
-
+      {
```

Comment:
/charges/:id that is missing, is handled differently on the API side. It is returning a 422 error, rather than a 404 error.